common/source/docs/common-compass-setup-advanced.rst

```diff
@@ -19,183 +19,68 @@ will not move in the correct direction in autopilot modes (i.e. AUTO,
 LOITER, PosHold, RTL, etc). This can lead to circling (aka
 "toiletbowling") or fly-aways.
 
-ArduPilot currently allows up to three compasses to be connected. Only
-one compass (specified using the ``COMPASS_PRIMARY`` parameter) is used
-for navigation. While many autopilots have an internal compass or compasses, many
-will instead use an external compass. This provides more reliable data
+ArduPilot multiple compasses to be connected, but only three are used at one time. And usually, only the primary compass is used, unless there are consistency problems with its readings versus other compasses and/or other sensors. In this case, the autopilot will automatically determine which of the first three compasses to use.
```

Comment:
I think we are missing "supports" or "allows" here.

Reply:
Is this true, I'm actually not sure if the EKF ever fails over to a 2nd compass.. perhaps it does..

Reply:
thats what I understood from Tridge....

Reply:
sometimes I have to backspace so often because I type dyslexically, my thoughts outpace the words! will add "allows"

Reply:
Ok, if Tridge says it is so then great

Reply:
added...thanks